### PR TITLE
Sept/Mid-Oct 2022

### PR DESCRIPTION
**Pull request by SethBarberee**
Rest of Pokemon found data is converted to JSON. Most of it is split to corresponding dungeons. The extra/non-used dungeons share tables which is interesting.

Code wise, it has been mainly Orb/Move action functions and continuing that here

### CODE REVIEW

File: include/dungeon_entity.h [hunk 1]
```diff
@@ -215,7 +215,8 @@ struct DungeonEntityData
     /* 0x16C */ struct Position targetPosition;
     /* 0x170 */ struct Position posPixel;
     u32 unk174;
-    u8 fill178[0x184 - 0x178];
+    u8 fill178[0x17A - 0x178];
+    u16 mimicMoveIDs[5]; // All moves that Mimic has copied (not sure on size...)
```
Comment (AnonymousRandomPerson) [inline include/dungeon_entity.h:219]: There are only 4 move slots, so the size is likely 4 (`MAX_MON_MOVES`), leaving an unknown after this array.

File: src/status_checker.c [hunk 2]
```diff
@@ -140,6 +239,966 @@ const u8 gDungeonCamouflageTypes[76] = {
...
+                iVar5 = 1;
+            }
+            sub_806F370(pokemon,pokemon,iVar5,0,0,0,sub_8057600(move,param_4),0,1,0);
+        }
+    }
+    return flag;
+}
+
+
+// NOTE: Either Excavate or Rock Smash (90% sure it's Rock Smash)
```
Comment (AnonymousRandomPerson) [inline src/status_checker.c:773]: Both moves are the same, so chances are that both of them call this function.

File: include/dungeon_entity.h (hunk 1 above)
Comment (SethBarberee) [reply include/dungeon_entity.h:219]: Testing locally, it aligns/matches with this and no unknown so will probably run with this
```suggestion
    u16 mimicMoveIDs[MAX_MON_MOVES]; // All moves that Mimic has copied (not sure on size...)
```

Comment (AnonymousRandomPerson) [reply include/dungeon_entity.h:219]: Oh nice, that's probably correct then.

File: src/status_checker.c (hunk 2 above)
Comment (SethBarberee) [reply src/status_checker.c:773]: Yeah, so keep the name then and delete the comment? Or do you prefer something different?

Comment (AnonymousRandomPerson) [reply src/status_checker.c:773]: Yeah, the comment can be removed.

**Review by AnonymousRandomPerson — APPROVED** [review]

**Pull request merged**